### ISSUE TITLE
pip to install opencv and opencv contrib

### ISSUE THREAD

**Issue by xymsh** [krematas/soccerontable#9]
It's trouble to build opencv and opencv contrib from source. There are hundreds of errors during compilation. I found a much easier way.

Actually, opencv in c++ is needed. So pip to install is not feasible.

**Comment by hysoka79** [comment]
But isn't the code run opencv in c++?
I'm now stucking at the problem that it couldn't include the opencv2/ximgproc

**Comment by xymsh** [comment]
The latest opencv and opencv contrib do not include ximgproc. You need to install opencv 3.1 and opencv contrib 3.1. Version 3.2 is OK too.

I solved it with 
`pip install opencv-python==3.2.0.7`
`pip install opencv-contrib-python==3.2.0.7`

**Comment by xymsh** [comment]
opencv and opencv contrib are run in python3, at least until the calibration phase. I now stuck at manual calibration phase. I have never succeeded in it.

By the way, I tried to build opencv and opencv contrib from source. But after I made soft link of cv2.so file, I cannot import it due to the following error. 
`ImportError: libz-a147dcb0.so.1.2.3: cannot open shared object file: No such file or directory`

I cannot solve it. So I used pip to install them instead.
